.github/workflows/prettier-eslint.yml

```diff
@@ -0,0 +1,40 @@
+---
+name: Lint and Format
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  lint-and-format:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: "16"
+
+      - name: Install Dependencies
+        run: npm install
+
+      - name: Run ESLint and Prettier
+        run: |
+          npm run lint -- --fix
+          npm run format -- --write
```

Comment:
no need for the extra options:
https://github.com/Dun-sin/Whisper/blob/c2dc613d32e1cfb41e11098db3bfe3cc19c2e723/client/package.json#L9-L10